### ISSUE TITLE
Provide option to ignore known_hosts file

### ISSUE THREAD

**Issue by mjbright** [davrodpin/mole#37]
## Affected Version
Latest commit (843dce7df3b3a7a0d88002c8b2f0f702ec5b9182)

## Expected Behavior
First try running on Windows 10 under cygwin I created an alias which I then try to use.
```mole -alias ldlc_minishift -remote 192.168.99.100:8443  -server mjb@192.168.0.40 -key $(cygpath -m ~/.ssh/id_rsa)```
```mole -start ldlc_minishift```

I expect connect to succeed.

## Current Behavior
mole fails to find known_hosts file because of horrible Windows pathnames!

> time="2018-10-22T18:59:30+02:00" level=info msg="listening on local address"
> local_address="127.0.0.1:28302"
> time="2018-10-22T19:02:19+02:00" level=error msg="error generating ssh client config: error while parsing 'known_hosts' file: C:\\tools\\cygwin\\home\\windo\\.ssh\\known_hosts: open C:\\tools\\cygwin\\home\\windo\\.ssh\\known_hosts: The system cannot find the path specified." tunnel="[local:127.0.0.1:28302, server:192.168.0.40:22, remote:192.168.99.100:8443]"

## Steps to Reproduce

1. On Windows/Cygwin: Create an alias with, for example
mole -alias ldlc_minishift -remote 192.168.99.100:8443  -server mjb@192.168.0.40 -key $(cygpath -m ~/.ssh/id_rsa)

2.On Windows/Cygwin: Start with that alias, for example
mole -start ldlc_minishift

Obviously this is a work-around for a Windows pathname problem but the --ignore-known-hosts option could be useful more generally.

A work around for the specific Windows pathname problem is welcome.

**Comment by davrodpin** [comment]
Hi @mjbright, I think we have two issues:

1. mole is not properly working on windows.
2. there should be an option to ignore the know_hosts files

Number 2 is being currently handled on #14.

**Comment by davrodpin** [comment]
and thanks for providing feedback! 😄

**Comment by davrodpin** [comment]
Insecure mode, which ignores the `known_hosts` file was implemented on #52

**Comment by davrodpin** [comment]
Support for windows implemented on #65.
It seems to be working through both `Command Line ` and `Cygwin`.

That's what I used to install mole on Cygwin:

```shell
$ curl -O -L https://github.com/davrodpin/mole/releases/download/v0.3.0/mole0.3.0.windows-amd64.zip && unzip mole0.3.0.windows-amd64.zip
```
